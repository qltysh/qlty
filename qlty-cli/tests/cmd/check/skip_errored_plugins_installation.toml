bin.name = "qlty"
args = ["check", "--all", "--no-cache", "--skip-errored-plugins"]
