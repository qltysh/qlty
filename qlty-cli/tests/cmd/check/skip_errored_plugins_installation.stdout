 ERRORS: 1 

Plugin        Result  Message                      Debug File
always_error  Error   Exited with code 1 in [..]s  .qlty/out/invoke-[..].yaml

Installation errors found:
Error installing tool gitleaks@8.24.2

